delocate/tools.py

```diff
@@ -262,7 +264,12 @@ def get_rpaths(filename):
     rpath : tuple
         rpath paths in `filename`
     """
-    out = back_tick(['otool', '-l', filename])
+    try:
+        out = back_tick(['otool', '-l', filename])
+    except RuntimeError:
```

Comment:
When I added `get_rpaths` before the `get_install_names` function in `tree_libs` I needed it to ignore non-binary files like `get_install_names` does.  This `try..except` clause can be moved to `tree_libs` if you like.